plugins/woocommerce-admin/client/customize-store/transitional/survey-form/index.tsx

```diff
@@ -98,12 +98,12 @@ export const SurveyForm = ( {
 		} else {
 			trackEvent( 'ces_feedback', {
 				action: 'customize_your_store_on_core_transitional_survey_complete',
-				rating,
+				score: rating,
 				choose_design_my_own_theme: isStreamlineChecked,
 				choose_dislike_themes: isDislikeThemesChecked,
 				choose_themes_not_match: isThemeNoMatchChecked,
 				choose_other: isOtherChecked,
-				feedback: feedbackText,
+				comments: feedbackText,
 				spill_beans: spillBeansText,
```

Comment:
This does mean that `spill_beans` won't be included in the FSOR.
May not a huge deal for now, but we could add an improvement to the FSOR to allow for `additional_comments` or something to that affect that would also allow us to import the above.
Or maybe we want to make this configurable through the JSON.

Reply:
What about the rest of the properties like `choose_design_my_own_theme`? Even if they do not show on FSOR, they are still available on our tracking infrastructure, right?

Reply:
I just checked and I can confirm they are still available :)

Reply:
Yes, everything is available in the tracking infrastructure